### PR TITLE
feat: add emoji support

### LINKED ISSUE #50: Feature: Add node emoji to highlight cards
### Type of feature

🍕 Feature

### Current behavior

Emojis are not being generated in the card preview. 

https://insights.opensauced.pizza/feed/171

Screen Shot 2023-06-06 at 10 59 06 AM

### Suggested solution

Use node-emoji wrapper for copy. We use it in open-sauced/hot

https://github.com/open-sauced/hot/pulls?q=is%3Apr+emoji+is%3Aclosed

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by a0m0rajab**
## Description

This PR added emoji support to the OpenGraph, at the beginning, it tried the proposed solution by @bdougie in using `node-emoji`, which it did not work, then it went to use the Satori emoji system, which solved the issue. The utils file were copied from `vercel/satori` repo without any change. 

I kept the file without change and had all the API options to raise a discussion about which emojis to choose, right now we are using `twemoji`, but there are a couple of different options which you can try from here: 

https://og-playground.vercel.app/
- go to advanced 
- change emoji options. 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [x] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

fixes #50 

## Mobile & Desktop Screenshots/Recordings

Before: 

after: 

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by a0m0rajab** [comment]
Emoji Styles: 
Twemoji: 
image
Fluent: 
image
Fluent Flat: 
image
Noto: 
image
Blob: 
image
Open Moji: 
image

**Comment by brandonroberts** [comment]
Nice work @a0m0rajab!

File: src/social-card/templates/highlight-card.template.ts [hunk 1]
```diff
@@ -15,7 +16,7 @@ const highlightCardTemplate = (
       <div tw="flex-row justify-between" style="gap: 16px;">
         <div tw="w-906px flex-col flex-nowrap" style="gap: -10px;">
           <h1 tw="text-72px leading-72px text-black tracking-tight" style="width: 926px; font-weight: 500;">
-               OpenSauced Highlight
+               OpenSauced Highlight
```
Comment (jpmcb) [inline src/social-card/templates/highlight-card.template.ts:19]: nit: Does this need a hanging space at the end of this line?

Comment (a0m0rajab) [reply src/social-card/templates/highlight-card.template.ts:19]: No, I had an emoji here to test it (When I did not find an highlight with emojis :) ) 

Would adding a pizza emoji be a good option here?

Comment (jpmcb) [reply src/social-card/templates/highlight-card.template.ts:19]: Sure! Good idea!! 🍕

Comment (a0m0rajab) [reply src/social-card/templates/highlight-card.template.ts]: Thank you, I just added an emoji at the beginning of this line! 🍕

**Pull request merged**